django/db/utils.py

```diff
@@ -102,6 +104,17 @@ def inner(*args, **kwargs):
         return inner
 
 
+def digest(*args):
```

Comment:
Can we find a better name for this. I know it's generating the hexdigest for all passed arguments, but that's still irritating to me.